package/src/libraries/react/ReactAutumnProvider.tsx

```diff
@@ -27,8 +27,9 @@ export const ReactAutumnProvider = ({
   betterAuthUrl,
   headers,
   convexApi,
+  pathPrefix,
 }: {
-  children: React.ReactNode;
+  children: any;
```

Comment:
```suggestion
  children: React.ReactNode;
```

could you bring back the type?